libdeno/binding.cc

```diff
@@ -382,6 +395,86 @@ bool ExecuteV8StringSource(v8::Local<v8::Context> context,
   return true;
 }
 
+void Eval(const v8::FunctionCallbackInfo<v8::Value>& args) {
+  v8::Isolate* isolate = args.GetIsolate();
+  DenoIsolate* d = FromIsolate(isolate);
+  v8::EscapableHandleScope handleScope(isolate);
+  auto context = d->context_.Get(isolate);
+  v8::Context::Scope context_scope(context);
+
+  CHECK(args[0]->IsString());
+  auto source = args[0].As<v8::String>();
+
+  auto output = v8::Array::New(isolate, 2);
+  /**
+   * output[0] = result
+   * output[1] = ErrorInfo | null
+   *   ErrorInfo = {
+   *     thrown: Error | any,
+   *     isNativeError: boolean,
+   *     isCompileError: boolean,
+   *   }
+   */
+
+  v8::TryCatch try_catch(isolate);
+
+  auto name = v8_str("<unknown>");
+  v8::ScriptOrigin origin(name);
+  auto script = v8::Script::Compile(context, source, &origin);
```

Comment:
All `libdeno.eval`s happen in the same context, while each `eval` creates a separate new context

Reply:
This is a good argument for it. I'm hesitating because I'm refactoring module resolution right now and I'm wondering if this can't somehow fit into the larger design rather than being a one-off function. More later. Sorry for the slowness.